Client/Frontend/Browser/Playlist/VideoPlayer/UI/VideoPlayer.swift

```diff
@@ -202,9 +223,160 @@ class VideoView: UIView, VideoTrackerBarDelegate {
     
     @objc
     private func onOverlayDoubleTapped(_ gestureRecognizer: UITapGestureRecognizer) {
-        delegate?.togglePlayerGravity(self)
+        // Originally double tap changed size of the video to fit the screen,
+        // now we will use this gesture to advance/go back by 15 secs.
+        // Perhaps we should add a button to the player's toolbar overlay for toggling gravity.
+        // delegate?.togglePlayerGravity(self)
+
+        // Advance or go back by 15 seconds
+        let tapLocation: CGPoint = gestureRecognizer.location(in: self)
+
+        // If we got a tap in 0width...0.4width (left part of the view).
+        if tapLocation.x > 0 && tapLocation.x < 0.4*self.bounds.size.width {
```

Comment:
Space between `0.4 * self.bounds.width` and also no need for `bounds.size.width`. Can just do `bounds.width`.